diff -Nur old/images/Makefile.am new/images/Makefile.am
--- old/images/Makefile.am	2003-06-30 07:25:56.000000000 +0000
+++ new/images/Makefile.am	2003-11-05 00:41:01.000000000 +0000
@@ -1,7 +1,7 @@
 # GPL (C) 1998 Andreas Tille
 #   tille@debian.org
 
-pixmapdir = $(prefix)/share/xteddy
+pixmapdir = @datadir@/pixmaps/xteddy
 
 pixmap_DATA = teddy.png xteddy.png xpenguin.png xduck.png xtuxxy.png \
               xalex.png xbobo.png xklitze.png xruessel.png xnamu.png xbrummi.png \
diff -Nur old/Makefile.am new/Makefile.am
--- old/Makefile.am	2003-06-30 06:50:08.000000000 +0000
+++ new/Makefile.am	2003-11-05 00:27:15.000000000 +0000
@@ -3,16 +3,19 @@
 bin_PROGRAMS = xteddy
 
 xteddy_SOURCES = xteddy.c
+
+DEFS = -DPIXMAP_PATH=\"@datadir@/pixmaps/xteddy\" -Wall
 LIBS = `imlib-config --libs`
+INCLUDES = `imlib-config --cflags`
+
 man_MANS = xteddy.6
-bin_SCRIPTS = xtoys xteddy_test
 
-DEFS = -DPIXMAP_PATH=\"@PIXMAP_PATH@\" -Wall
+bin_SCRIPTS = xtoys xteddy_test
 
 SUBDIRS = html images
-EXTRA_DIST = $(man_MANS)  $(bin_SCRIPTS) \
-             README xteddy.README xtuxxy.credit \
-	     configure autogen.sh              
+
+EXTRA_DIST = README xteddy.README xtuuxxy.credit \
+	     configure autogen.sh
  
 
 
